docs/USAGE.rst

```diff
@@ -0,0 +1,366 @@
+
+.. image:: screenshots/export_buttons_highlighted.png
+   :height: 100
+   :width: 200
+   :alt: export buttons
+
+Recent exports are any items in the ``Approved`` folder that have not been exported before. After each export, items are tagged with metadata indicating that they have been exported.
+
+After export, a message is shown indicating how many files were exported, how many were already present (based on having the same file name) and the same file size, and how many were already present and differed in file size.
+
+
+Redaction
+=========
+
+At a high level in the Redaction process, the user can inspect the image and metadata for PHI/PII, and can indicate if any of these should be redacted. Redaction can occur from the ``Available To Process`` or ``Quarantined`` folder. Below are the specific steps to follow for the Redaction process. When a field is redacted so that it is changed, the original value in that redacted data field is completely replaced with the new value. When a field or image is redacted completely, it is removed and will not be available in the future.
+
+1. Reviewing the image(s) and metadata for PHI/PII.
+
+  1.1 Open the WSI file, either by clicking on a specific Item or clicking the ``Next Item`` action.
+
+  1.2 Review the image(s) and metadata fields for PHI/PII.
+
+    1.2.1 Review the image. If the low-resolution image at the top of the screen (see screenshot below) contains PHI/PII, then click XXX. If there is no PHI/PII, then go on to the next step. You may zoom and pan in the low-resolution image to see areas of the image in higher resolution.
+
+    TODO: what to do in this case? Reject?
+
+    1.2.2 Review metadata for PHI/PII, by scrolling down below the main image to the metadata section display. Some metadata fields will be automatically pre-redacted upon import, including titles and dates that are specific to each scanner manufacturer. See the Business Rules for WSI DeID section below for details.
+
+    The user can view metadata fields, and if any of these contain PHI/PII, the user should click the ``redact`` action for that field. The ``redact`` action will then have a line through the text, indicating that the field will be redacted (it has been marked for redaction).
+
+    1.2.3 Scroll down to the bottom of the screen and review the associated images (label, macro, and thumbnail).  If you see PHI/PII in individual associated images, redact and process them.
```

Comment:
This should end `images, select the classification of PHI in the image from the redact control.  The image will show an X through it to indicate that it will be redacted.`